### ISSUE TITLE
pytorch '1.8.0+cu111' x = torch.stack(x, dim=0) 

### ISSUE THREAD

**Issue by deahhh** [pytorch/pytorch#54134]
## To Reproduce

1. model = torchvision.models.resnet34(pretrained=True)
2. trans = transforms.Compose([
        transforms.ToPILImage(),
        transforms.Resize((224, 224)),
        transforms.ToTensor(),
        transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
    ])
3. x = [torch.rand(3, 224, 224)]
4. model.to("cuda").eval()
5. y1 = model(x[0].view(1, 3, 224, 224).to("cuda"))
6. y2 = model(torch.stack(x, dim=0).to("cuda"))
7. (y1 == y2 ) == False
8. actually, i got (y1==y2)==True in docker which pull from pytorch/pytorch   1.6.0-cuda10.1-cudnn7-runtime

cc @ngimel

**Comment by agolynski** [comment]
Could you be more specific what is the issue in question here?

**Comment by deahhh** [comment]
> @deahhh Could you be more specific what is the issue in question here?

@agolynski 
Thanks!
1~6 step by step, I got 7 in pytorch1.8.0+cu111', while, got 8, in pytorch/pytorch 1.6.0-cuda10.1-cudnn7-runtime, and the later is what I want.
Simply, pytorch 1.8.0 has one bug, using torch.stack and feeding result to resnet34 and I get wrong output, even stack one length list. tensor.View won't.
